### ISSUE TITLE
Error in performing [SCALE, GRID, HEADERS,....

### ISSUE THREAD

**Issue by museflip** [Audiveris/audiveris#221]
Hello,
Thank you very much Hervé and the whole team for this terrific job.

I can use Windows installer V5.1rc02 ( 9fa751d) released by released by Bacchushlg on 13 Aug.
 V5.1rc02 can generate a MusicXML file available for Musescore.

New official release Major user-oriented features (4f234fe) stops with an error before it can achieve the musicXML file.
The specific error message is 
[Je_suis_dAllemagne] Error in performing [SCALE, GRID, HEADERS, STEM_SEEDS, BEAMS, LEDGERS, HEADS, STEMS, REDUCTION, CUE_BEAMS, TEXTS, MEASURES, CHORDS, CURVES, SYMBOLS, LINKS, RHYTHMS, PAGE] java.util.concurrent.ExecutionException: java.lang.NullPointerException

Notes : 
1) Using clone git + gradlew.bat method leads to the same behaviour.
2) I am using Windows 7 OS and I had already installed Tesseract 4.0 for other purposes
3) By the way, Tesseract 4.0 (Mannheim university packages) that I usually use even when it was in RC releases and even alpha releases is far more efficient than 3.04 release and quite reliable (mostly used for french language)

Please see attach simple example which leads to the error.

In conclusion : 
* Windows installer V5.1rc02 ( 9fa751d) OK
* Major user-oriented features (4f234fe) KO
* clone git + gradlew.bat                        KO

Merry Christmas !

![je_suis_dallemagne](https://user-images.githubusercontent.com/46139401/50443602-61646780-0904-11e9-82b6-dad0c0cc5e76.jpg)

**Comment by hbitteur** [comment]
@museflip 
Regarding Tesseract, I thought current Audiveris was not yet compatible with Tesseract 4.0, could you share the modifications you had to make (I suppose they were limited to build.gradle file)
This would help us.

**Comment by museflip** [comment]
@hbitteur 
Thanks Hervé for your replies.
Thanks to the hint you gave me about Tesseract, I uninstalled Tesseract 4.0.
Audiveris 5.1.0 is now working fine on my environment.
The conclusion is that Audiveris 5.1.0 is not compatible with a Tesseract 4.0 installation, whether you install Tesseract before or after Audiveris.
Curiously, V5.1rc02 can cohabitate with Tesseract 4.0 despite the warning it raises about Tesseract during the process.

Do you still need the complete log file you requested from me ? I can provide it to you.
Do not hesitate to ask me further information if you find it valuable and if I am able to get it for you.

**Comment by museflip** [comment]
@hbitteur
To answer your second question, 
(could you share the modifications you had to make (I suppose they were limited to build.gradle file) ?)
I did not connect Audiveris with Tesseract 4.0.
On one hand, I had already Tesseract 4.0 installed and on the other hand, I installed Audiveris 5.1.0.
During the build phase, the only problem I experienced is that I had a complete jdk-8u144 but only a jre-8u191. I solved the compilation issue as soon as I installed a complete jdk for 8u191 too.
Thanks for your help.

**Comment by hbitteur** [comment]
@museflip 
Required Tesseract binary version is specified in build.gradle file as '3.04.01-1.3'
However, this does not automatically apply on Tesseract data (language) files.

I just double-checked the installation/building instructions in Audiveris handbook. I thought we used to call user attention on Tesseract 3 vs 4, at least in the initial README.md file on audiveris repository.
Anyway, I'm updating the handbook accordingly.
 
Thanks  for pointing this out.
Closing this issue now
